### ISSUE TITLE
The Directus CLI cryptically silences fatal errors

### ISSUE THREAD

**Issue by nathanvogel** [directus/directus#3761]
Hi,

### What I expected to happen: 
- Fatal errors in the CLI are logged to the console. 

### What actually happens:
- The CLI stays completely silent.

### How to reproduce: 

Setup a project on a host where a Directus CLI dependency has a fatal bug. 
For example in my case it was this one, https://github.com/ranisalt/node-argon2/issues/276

```
npx create-directus-project my-project
cd my-project
```

And then running any `directus` or `npx directus` or `node_modules/.bin/directus` etc command fails silently. For example: 
```
nodejs@node53009 /home/user/test-directus $ npx directus -v
nodejs@node53009 /home/user/test-directus $ 
```

So I edited `node_modules/directus/cli.js` from: 

```
try {
return require('./dist/cli/index.js');
} catch {}
```

to:

```
try {
return require('./dist/cli/index.js');
} catch(e) { console.log(e); }
```

and now it prints this error, which is the behavior I would expect, making it instantly clear why Directus was **not responding at all**:

```
nodejs@node53009 /home/user/test-directus $ npx directus -v
Error: /lib64/libstdc++.so.6: version `CXXABI_1.3.8' not found (required by /home/user/test-directus/node_modules/argon2/lib/binding/napi-v3/argon2.node)
    at Object.Module._extensions..node (internal/modules/cjs/loader.js:1122:18)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (/home/user/test-directus/node_modules/argon2/argon2.js:9:56)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
nodejs@node53009 /home/user/test-directus $ 
```

And after fixing the error (rebuilding as indicated in the linked bug report), the Directus CLI works as expect.

Hope you can make this clearer! Thank you!

**Comment by duplich** [comment]
I had the same issue, and it is super helpful to know what happens instead of failing silently and then stand in wonder. I have been struggling for hours to make directus run locally with remote database (postgres on DigitalOcean). I came to this and I was in shock that you guys in Directus have such approach... Please improve the error handling with useful error logs.

**Comment by tobius** [comment]
>

@duplich This particular issue was resolved and closed almost two years ago, but there is a more recent issue that might be related to what you're experiencing right now: https://github.com/directus/directus/issues/16284

The good news is this is already resolved: https://github.com/directus/directus/pull/16299

The bad news is this hasn't been deployed yet. I think it will be included in `9.21.3`?

In any event, if this is what you're experiencing today you can employ a temporary workaround:
```bash
export LOG_STYLE=raw
```

This won't print pretty logs, but it will print the logs that you're missing.

_p.s. This is what my team is doing in the interim._

**Comment by tobius** [comment]
> The bad news is this hasn't been deployed yet. I think it will be included in `9.21.3`?

#16299 is available in `9.22.0`

**Comment by duplich** [comment]
Thanks for reaching out @tobius. I have given up for now, trying to do the instalment of Directus locally. But in the future I was advised to install the directus with local sqlite version, and after the instalment is done, change the database connection to remove db on postgres. This is how I ran into the issue with missing error logs in the first place. I will test that approach and hope to run into no issues nor missing error logs :). Thanks for the quick response.
